libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3744,6 +3760,11 @@ MAV_RESULT GCS_MAVLINK::handle_command_long_packet(const mavlink_command_long_t
         result = handle_command_do_fence_enable(packet);
         break;
 
+#define MAV_CMD_DO_SCALE_WITH_ZOOM 40003
```

Comment:
This will need to be added to the `MAV_CMD` enum within the `mavlink` submodule's `.xml` definitions.  You can extend the enum in `common.xml`, `ardupilot.xml` or even `planckaero.xml`.  Additionally, there will need to be an additional pull request for the `mavlink` repo to add this change, and this PR should point to the updated submodule commit.